### PR TITLE
chore: Add test USDC asset to E2E setup

### PR DESCRIPTION
**Pull request by freddyli7**
Add test USDC asset creation, metadata setup and mint to e2e test setup

The illustration of relationship of the `asset fee, customized asset and asset MultiLocation and resourceID`
image

### CODE REVIEW

File: scripts/js/setup.js [hunk 1]
```diff
@@ -90,12 +193,44 @@ async function main() {
...
+    const nativeMinBalance = 100;
+    const nativeName = "Native test asset";
+    const nativeSymbol = "NATIVE";
+    const nativeDecimal = 12;
+    await createAsset(api, nativeAssetID, nativeAdmin, nativeMinBalance, true, sudo);
+    await setAssetMetadata(api, nativeAssetID, nativeName, nativeSymbol, nativeDecimal, true, sudo);
+    await mintAsset(api, nativeAssetID, nativeAdmin, 100000000000000, true, sudo); // mint 100 native to Alice
+
+    const nativeAsset = 0; // TODO: I suppose 0 means `Concrete(MultiLocation::new(0, Here);` ?
+                           // TODO: and how to link this nativeAsset with NativeLocation: MultiLocation that defined in the runtime?
```
Comment (freddyli7) [inline scripts/js/setup.js:214]: @tolak please check out my questions here

File: scripts/js/setup.js [hunk 2]
```diff
@@ -90,12 +193,44 @@ async function main() {
...
+    const usdcMinBalance = 100;
+    const usdcName = "USDC test asset";
+    const usdcSymbol = "USDC";
+    const usdcDecimal = 12;
+    await createAsset(api, usdcAssetID, usdcAdmin, usdcMinBalance, true, sudo);
+    await setAssetMetadata(api, usdcAssetID, usdcName, usdcSymbol, usdcDecimal, true, sudo);
+    await mintAsset(api, usdcAssetID, usdcAdmin, 100000000000000, true, sudo); // mint 100 USDC to Alice
+
+    const usdcAsset = 1; // TODO: how to assign usdcAsset param here to represent the UsdcLocation: MultiLocation that defined in the runtime?
+                        //  TODO: and how to link this created usdcAsset with UsdcLocation: MultiLocation that defined in the runtime?
```
Comment (freddyli7) [inline scripts/js/setup.js:231]: @tolak please check out my questions here as well

File: bridge/src/mock.rs [hunk 3]
```diff
@@ -180,7 +180,7 @@ parameter_types! {
 	pub RelayNetwork: NetworkId = NetworkId::Polkadot;
 	pub AssetsPalletLocation: MultiLocation =
 		PalletInstance(<Assets as PalletInfoAccess>::index() as u8).into();
-	pub PhaLocation: MultiLocation = MultiLocation::here();
+	pub NativeLocation: MultiLocation = MultiLocation::here();
```
Comment (tolak) [inline bridge/src/mock.rs:183]: Suggest renaming to `NativeAssetLocation`, ResourceId the same

**Review by tolak — APPROVED** [review]
LGTM

**Pull request merged**